selene_sdk/samplers/random_positions_sampler.py

```diff
@@ -168,6 +173,12 @@ def __init__(self,
         for mode in self.modes:
             self._update_randcache(mode=mode)
 
+        self.replacement = replacement
+        self.sample_size = sample_size
+
+        self.tot_sample = np.zeros(1) # will be 1 indexed.
```

Comment:
why is this 1-indexed? also I'm confused about the `tot_sample` data structure in general, can you document how large it is expected to be

Reply:
`tot_sample` should be initialized to `tot_sample = np.zeros(sample_size)` so that it's already a vector of zeros of length `sample_size`